tasks/main.yml

```diff
@@ -44,6 +44,26 @@
     update_cache: yes
     state: installed
 
+- name: grep for LimitNOFILE in systemd service
```

Comment:
My god! I did not expect there to be an official module for inis! I changed it to this, which is waaaaay better:

```
- name: Set the file descriptor limit to a high value in the systemd service
  ini_file:
    path: /lib/systemd/system/rabbitmq-server.service
    section: Service
    option: "{{ item }}"
    value: 65535
    backup: yes
  with_items:
    - LimitNOFILE
    - LimitNPROC
```